content/pt/docs/referencia-da-api/pix/recebimento/atualizar-qrcode-estatico/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Atualizar QRCode Estático"
+linkTitle: "Atualizar QRCode Estático"
+date: 2021-08-04T15:19:30-03:00
+lastmod: 2021-08-04T15:19:30-03:00
+draft: false
+weight: 4
+description: >
+  
+---
+<br>
+Através desse endpoint será possível adicionar informações no QRCode estático criado.
+<br>
+<br>
+
+```
+PATCH https://sandbox-api.openbank.stone.com.br/api/v1/pix_payment_static_invoices/{{id}}
+```
+<br>
+
+##### **HEADERS**
+---
+
+**authorization*** `string`
+<br> Bearer token de autenticação.
+
+**x-stone-idempotency-key*** `string`
+<br> Chave de idempotência.
+
+<br>
+
+##### **PATCH PARAMS**
+
+**id** `uuid`
+<br> Identificador do QRCode criado
+
+##### **BODY PARAMS**
+---
+<br>
+
+**additional_information** `string`
+
+<br>
+
+##### **Response**
+---
+
+```
+200 OK
+```
+
+```json
+{
+  "id": "<uuid>",
+  "account_id": "<uuid>",
+  "participant_ispb": "<string>",
+  "key": "<string>",
+  "key_type": "<string>",
+  "transaction_id": "<string>",
+  "amount": "<integer>",
+  "additional_information": "<string>",  
+  "request_id": "<string>",
+  "created_at": "<iso8631>",
+  "updated_at": "<iso8631>",
```

Comment:
well, I'm not pretty sure about this. But the ISO number for date-time is 8601, isn't it? also, I believe a date-time prefix would stand clear for the field.

Reply:
agreed!